tensorboard/plugins/mesh/metadata_test.py

```diff
@@ -69,27 +82,40 @@ def test_parse_plugin_metadata(self):
                      parsed_metadata.content_type)
     self.assertEqual(self.shape, parsed_metadata.shape)
     self.assertEqual(self.json_config, parsed_metadata.json_config)
+    self.assertEqual(self.components, parsed_metadata.components)
 
   def test_metadata_version(self):
     """Tests that only the latest version of metadata is supported."""
-    self._create_metadata()
-    # Change the version.
     with patch.object(metadata, 'get_current_version', return_value=100):
+      self._create_metadata()
+    # Change the version.
+    with patch.object(metadata, 'get_current_version', return_value=1):
```

Comment:
Given that this PR doesn’t actually bump the metadata version, we can
probably revert the changes to this test, yes?

Reply:
But it changed the logic of version validation, so I think this test is still valid.